src/coreclr/tools/dotnet-pgo/MethodMemoryMap.cs

```diff
@@ -20,15 +21,38 @@ internal class MethodMemoryMap
         private readonly ulong[] _infoKeys;
         public readonly MemoryRegionInfo[] _infos;
 
+        struct JittedID : IEquatable<JittedID>
```

Comment:
I guess the whole thing can be replace with [record struct](https://sharplab.io/#v2:EYLgtghglgdgPgWAFACcCmBjA9igJgAgGcAXFAVw2PwCkpji1cBJAEQAoAbLGAc3wFk0xABZZmLADT4uvfACU01JgBVWASgDcQA=) if we allowed to use C# 10 in dotnet-pgo 🙂

Reply:
Probably. I'm still stuck in the C# language I learned years ago, and I forget about all the new stuff.